app/views/profile/profile.html.erb

```diff
@@ -0,0 +1,153 @@
+<link href="//maxcdn.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
```

Comment:
So bindet man kein Bootstrap in Rails ein. Das muss über Gems und Config-Files erfolgen. Dazu mehr in der Beispielanwendung der Vorlesung und unter https://rubygems.org/gems/bootstrap